pkg/controller/argocd/util.go

```diff
@@ -186,6 +192,10 @@ func (r *ReconcileArgoCD) getArgoServerURI(cr *argoprojv1a1.ArgoCD) string {
 		}
 	}
 
+	if _, err := url.ParseRequestURI(host); err == nil {
+		return host
```

Comment:
We should probably log something here to warn that we're ignoring the URL, otherwise it might be a bit confusing that a "valid" looking YAML didn't work, and no error messages?

Reply:
like we discussed, i think it's OK to not have this check, since it says hostname rather than a URL. So we don't need to have this check !! 
worst case if a user provides a URL rather than a FQDN, url in the config map would end up something like `https://https://xyz.com`